### DISCUSSION TITLE
10번째 미팅 때 논의하고 싶은 내용

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [Joseph-Cha/JustFight_Client#48] [Q&A] [answered]
### 세부 내용
 1. PR 리뷰
    - #39 (플레이어 생성 설계 문서)
    - #42 (플레이어 생성 설계문서 수정)
    - #44 (플레이어 생성 구현)
    - #45 (플레이어 이동 설계 문서)

2. MVC 모델 리뷰
    - [링크](https://github.com/Joseph-Cha/JustFight_Client/pull/44#discussion_r622207754)
    - Player UI 파트와 데이터 처리 부분을 나눠서 작업 진행 중
    - Q? Player 이동을 구현할 때 위 모델을 어떻게 적용하면 좋을지?

거의 2달 만에 리뷰라 기억이 가물가물한 것 같습니다..ㅋㅋ
그래도 그동안 쌓인(?) PR들이 있어서 간단히 짚고 넘어가려고 합니다.

그리고 리뷰 중 말씀해 주신 MVC 모델에 대해서 이야기 나누면서
향후 어떻게 작업하면 좋을지도 이야기하면 좋을 것 같습니다.

**Comment by jongfeel** [comment, marked as answer]
MVVM에서 핵심적인 걸 보고 싶다면

``` csharp
INotiryPropertyChanged
```
